web/src/routes/(user)/people/+page.svelte

```diff
@@ -324,6 +362,7 @@
             <PeopleCard
               {person}
               on:change-name={handleChangeName}
+              on:set-birth-date={handleSetBirthDate}
```

Comment:
You could also just do to avoid using the `CustomEvent<T>` interface in the handlers. It's cleaner IMO.

```suggestion
              on:set-birth-date={() => handleSetBirthDate(person)}
```